manifests/init.pp

```diff
@@ -355,9 +357,12 @@
 
 
 class sensu (
-  $version                        = 'latest',
+  $version                        = 'installed',
   $sensu_plugin_name              = 'sensu-plugin',
-  $sensu_plugin_provider          = undef,
+  $sensu_plugin_provider          = $::osfamily ? {
+    'windows' => 'gem',
+    default   => undef,
+  },
```

Comment:
This avoids `Provider sensu_gem is not functional` on windows.  Considered moving the behavior to package.pp, put it here to make it clear.